packages/terra-status-view/src/StatusView.jsx

```diff
@@ -164,7 +164,7 @@ const StatusView = ({
   return (
     <div {...customProps} className={outerViewClassNames}>
```

Comment:
can we get role and aria attribute to be added at top div to ensure that they are not overridden by custom props. !!

Reply:
nit: role should be added before the Aria
```js
role="alert" aria-live="polite"
```